src/common/communication.ts

```diff
@@ -167,3 +164,7 @@ export const UPDATE_TAB_ORDER = 'update-tab-order';
 export const GET_LAST_ACTIVE = 'get-last-active';
 export const GET_ORDERED_SERVERS = 'get-ordered-servers';
 export const GET_ORDERED_TABS_FOR_SERVER = 'get-ordered-tabs-for-server';
+
+export const UPDATE_APPSTATE = 'update-appstate';
+export const UPDATE_APPSTATE_TOTALS = 'update-appstate-totals';
+export const UPDATE_APPSTATE_FOR_VIEWID = 'update-appstate-for-viewid';
```

Comment:
I think this should be `UPDATE_APPSTATE_FOR_VIEW_ID` since they're two separate words. I also misread it as "for viewed" at first